meta-ibm/recipes-phosphor/configuration/platform-fru-detect_git.bb

```diff
@@ -0,0 +1,17 @@
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+SRC_URI = "git://git@github.com/amboar/platform-fru-detect.git;protocol=ssh;branch=main"
```

Comment:
Shouldn't this also be in ibm-openbmc?  I'm not sure how  it can follow the review process otherwise?  Or is this just a temporary thing that you are going to be doing a different  way?

Reply:
Hosting it in my github namespace is temporary. The whole project is something I developed while working to support NVMe drive temperatures. The aim is to get it added under the openbmc org, at which point we could fork it into ibm-openbmc if necessary. I've started discussions justifying it on discord, trying to convince Ed that it's useful but so far haven't got a lot of feedback.